gettingstarted.md

```diff
@@ -60,6 +60,28 @@ If you use Eclipse, you will need to install the Maven plugin for Eclipse: [ecli
 
 Michael Depies has written this guide to [installing Deeplearning4j on Eclipse](https://depiesml.wordpress.com/2015/08/26/dl4j-gettingstarted/).
 
+## <a name="eclipse_mvn_scm_checkout">Using DL4J Examples in Eclipse with direct checkout from GIT via the Maven plugin</a> 
+
+If you just want to get the examples running within Eclipse and without using command line at all you can also directly checkout the project from SCM:
```

Comment:
@huitseeker what do you mean by "expand "from Eclipse's built-in Source Control Management (SCM):" once" ?